problems/3-planet-distance/index.js

```diff
@@ -1 +1,75 @@
 'use strict';
+
+var obj = {
+  'mercury_venus': 0.33,
+  'mercury_earth': 0.61,
+  'mercury_mars': 1.13,
+  'mercury_jupiter': 4.81,
+  'mercury_saturn': 9.15,
+  'mercury_uranus': 18.75,
+  'mercury_neptune': 29.67,
+  'venus_mercury': 0.33,
+  'venus_earth': 0.28,
+  'venus_mars': 0.80,
+  'venus_jupiter': 4.48,
+  'venus_saturn': 8.82,
+  'venus_uranus': 18.42,
+  'venus_neptune': 29.34,
+  'earth_mercury': 0.61,
+  'earth_venus': 0.28,
+  'earth_mars': 0.52,
+  'earth_jupiter': 4.20,
+  'earth_saturn': 8.54,
+  'earth_uranus': 18.14,
+  'earth_neptune': 29.06,
+  'mars_mercury': 1.13,
+  'mars_venus': 0.80,
+  'mars_earth': 0.52,
+  'mars_jupiter': 3.68,
+  'mars_saturn': 8.02,
+  'mars_uranus': 17.62,
+  'mars_neptune': 28.54,
+  'jupiter_mercury': 4.81,
+  'jupiter_venus': 4.48,
+  'jupiter_earth': 4.20,
+  'jupiter_mars': 3.68,
+  'jupiter_saturn': 4.34,
+  'jupiter_uranus': 13.94,
+  'jupiter_neptune': 24.86,
+  'saturn_mercury': 9.15,
+  'saturn_venus': 8.82,
+  'saturn_earth': 8.54,
+  'saturn_mars': 8.02,
+  'saturn_jupiter': 4.34,
+  'saturn_uranus': 13.94,
```

Comment:
Looks like you have this represented twice, with two different values.

`'saturn_uranus': 13.94,` AND `'uranus_saturn': 9.60,`

I would check this value mapping.  You should only need a single item per planet combination